### ISSUE TITLE
[Suggestion]: Corpses mod compatibility

### ISSUE THREAD

**Issue by kaka34665** [nonamecrackers2/crackers-wither-storm-mod#2205]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Normally with the corpses mod, when the wither storm chomps you, your corpse just falls down to the ground. Seems kinda off isn't it? So what I'm suggesting is making a config that corpses will not appear when you get chomped by the wither storm, as your items are put inside a phlegm block by the beast anyway. Maybe another config can be added that the wither storm can pick up and consume your corpse, which it of course either your items will be in a phlegm block or a sicked version of your corpse that can be found in the bowels or when the wither storm is killed but only if it ends up straight into it's mass.

**Comment by TaintedBlock** [comment]
I feel like the corpse should stay as it is now since it basically is just acts as a phlegm block

**Comment by kaka34665** [comment]
> I feel like the corpse should stay as it is now since it basically is just acts as a phlegm block

So would it not work if there are codes that prevents the corpse from appearing when you get chomped and then your items will be in a phlegm block instead? I wonder if my suggestion on the wither storm sucking up your corpse would also work..

**Comment by nonamecrackers2** [comment]
I don't think this is possible because corpse mods override and clear your drops (which get stored in a corpse) before CWSM can detect them and spawn a phlegm block. I'll take a look into it, but the current order of priority is keepInventory, corpse mods, then the phlegm gravestone system.

**Comment by kaka34665** [comment]
> I don't think this is possible because corpse mods override and clear your drops (which get stored in a corpse) before CWSM can detect them and spawn a phlegm block. I'll take a look into it, but the current order of priority is keepInventory, corpse mods, then the phlegm gravestone system.

Can you let me know when it's possible? By the way I've also been wondering if it would be really cool if there's a config that allows you to turn on colliding with block clusters. Similarly to Solid Mobs mod.. I would find it really cool about the fact that you'll be able to ride block clusters before they get eaten by the wither storm.

**Comment by Arceus-77** [comment]
@kaka34665 I agree for a different reason, I want to ride the wither storm in creative, and get a birds eye view of the world(from atop the wither storm)

**Comment by Arceus-77** [comment]
> > I don't think this is possible because corpse mods override and clear your drops (which get stored in a corpse) before CWSM can detect them and spawn a phlegm block. I'll take a look into it, but the current order of priority is keepInventory, corpse mods, then the phlegm gravestone system.
> 
> Can you let me know when it's possible? By the way I've also been wondering if it would be really cool if there's a config that allows you to turn on colliding with block clusters. Similarly to Solid Mobs mod.. I would find it really cool about the fact that you'll be able to ride block clusters before they get eaten by the wither storm.

I agree to the bit about riding entites, I want to ride the wither storm in creative.
